app/frontend/wallet/cardano-wallet.ts

```diff
@@ -158,12 +133,7 @@ const CardanoWallet = (options) => {
 
   async function submitTx(signedTx) {
     const {txBody, txHash} = signedTx
-    const response = await blockchainExplorer.submitTxRaw(txHash, txBody).catch((e) => {
-      debugLog(e)
-      throw e
-    })
-
-    return response
+    return await blockchainExplorer.submitTxRaw(txHash, txBody)
```

Comment:
I would use type annotation here to make clear the returned value is a promise instead of awaiting on return